PHPGraphQLArticle.md

```diff
@@ -82,15 +82,13 @@ query UserPosts{
 }
 ```
 
-This query describes the arguments sent, and also the fields that are required. It's a lot clearer what information will be returned from the server.
+The above query describes the arguments sent, the required fields and is a lot clearer on what information will be returned from the server
 
-END OF JP REVISION 1
+GraphQL usually has only one endpoint and we make all requests there with queries like this. To find out what queries are available, we define a schema on the server. The server defines a schema that list the queries, it's possible arguments and return types. 
```

Comment:
To expose available queries, we define a schema on the server. The schema lists the queries, its possible arguments and the return types.